### PR TITLE
Advocate using ko in developer docs

### PR DESCRIPTION
**Pull request by a-roberts**
This is for https://github.com/tektoncd/dashboard/issues/44

I want us to be consistent with how tektoncd/pipeline is built and worked on so please do take a look, poking @vdemeester @jessm12 @dibbles and @vtereso 👼 , lemme know if there's something I've missed or messed up!

### CODE REVIEW

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -54,6 +56,44 @@ _Adding the `upstream` remote sets you up nicely for regularly
 - We've had good success using Docker Desktop: ensure your Kubernetes cluster is healthy and you have plenty of disk space allocated as PVs will be created for PipelineRuns.
 - Ensure you can push images to a Docker registry - the above listed requirements are only for local development, otherwise we pull in the tooling for you in the image.
 
+1. `GOPATH`: If you don't have one, simply pick a directory and add
+   `export GOPATH=...`
+1. `$GOPATH/bin` on `PATH`: This is so that tooling installed via `go get` will
+   work properly.
+1. `KO_DOCKER_REPO`: The docker repository to which developer images should be
```
Comment (vdemeester) [inline DEVELOPMENT.md]: There is also the `ko.local` hack (if you use `minikube` or docker-for-mac) as it doesn't require you to push to a registry, it directly load the images on the k8s docker daemon.
Can be added in a follow-up though :wink:

Comment (a-roberts) [reply DEVELOPMENT.md]: Mb, I only pushed the docs up on my first commit, addressed now

Good point, I remember using `ko.local` and advocated for it when building tektoncd/pipeline, I'm happy for that to be done as a follow-up also (we personally stick to using dockerhub so we needn't spin up and manage our own registry)

File: webpack.dev.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 const merge = require('webpack-merge');
 
 const common = require('./webpack.common.js');
-const { API_DOMAIN, PORT } = require('./config/config.json');
+const { API_DOMAIN, PORT } = require('./config-frontend/config.json');
```
Comment (vdemeester) [inline webpack.dev.js]: Isn't it `config_frontend` ? (looking at the folder)

**Comment by a-roberts** [comment]
@AlanGreene FYI as I've renamed the config folder, lemme know if this is cool with you too from a frontend perspective and then I'll merge this

**Comment by a-roberts** [comment]
@AlanGreene thanks, updated and noticed a ref in DEVELOPMENT.md too, lemme know if I've missed any others please, I've done an `npm install` and an `npm test` too

**Comment by AlanGreene** [comment]
Verified the coverage reports and running the client locally too. Looks good 👍

**Pull request merged**